crates/fuel-core/src/state/rocks_db.rs

```diff
@@ -217,15 +218,17 @@ impl KeyValueStore for RocksDb {
             }
             (Some(prefix), None) => {
                 // start iterating in a certain direction within the keyspace
-                let iter_mode = IteratorMode::From(prefix, direction.into());
+                let iter_mode =
+                    IteratorMode::From(prefix, DatabaseIterDirection(direction).into());
```

Comment:
If you use `DatabaseIterDirection` only here, it is better not to create this type and replace converting by some funciton=)